homeassistant/components/time_date/__init__.py

```diff
@@ -1 +1,21 @@
 """The time_date component."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_DISPLAY_OPTIONS, PLATFORMS
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Time & Date from a config entry."""
+    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
+    if entry.data.get(CONF_DISPLAY_OPTIONS):
+        # Clean data after first run.
+        hass.config_entries.async_update_entry(entry, data={})
```

Comment:
What's happening here?

Reply:
oh... tired... it has nothing to do with it. I added an abort_entries_match in the configuration flow